scripts/fast-reboot

```diff
@@ -69,7 +73,7 @@ sonic_asic_type=$(sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type)
 function clear_warm_boot()
 {
     debug "Failure ($?) cleanup ..."
-    config warm_restart disable || /bin/true
+    timeout 10 config warm_restart disable || /bin/true
```

Comment:
If timeout happens (check the exit status), should we bail out from the process?

Reply:
If we couldn't figure out the timeout, then we likely have to add more "timeout x" to all db calls, for now, I don't really want to litter the script with that.

Reply:
I think if we time-out for this configuration, we should not continue,  the script should exist with some reason code and print an error messages.  At this point, human intervention would be required.

Reply:
I thought about breaking out. But I was a bit concerned about if the DB issue was transient. With this change. At least we guaranteed to end execution in 60 second. I will have more information to have decision when the DB issue is RCA'ed. For instance, we could have limited retry count, maybe as you said, no retry at all.

Because at this time, we have already killed bgp, pre shutdown syncd, if there is a chance to move forward, I think we should. With that said, I agree with you thought of ending soon to get next level intervention.

BTW, to answer your other concern of running longer than 60 seconds. I adjusted the counter increment, so the worst case we could execute 65 second.

Reply:
Looks like we hit a docker bug and killed the database docker. I found that people hitting this bug with docker 1.12.6. And we are still running 1.11.1.

With this information. I think limited retry would be ok. Are you ok with 3 retries and no more than 65 seconds?

Reply:
Found this discussion: https://github.com/coreos/bugs/issues/1809. The conclusion is that it is not centos bug, rather a docker bug.

Reply:
As far as I can tell, the db container hang. The flag won't do anything at this point. I am still trying to track down how the container hang.

Reply:
Limited retry to 3 times.